package.json

```diff
@@ -2,6 +2,7 @@
   "name": "cortex",
   "version": "0.1.0",
   "private": true,
+  "homepage": "https://sirjoshi.github.io/Cortex-Command-Community.github.io/",
```

Comment:
It's not a big deal, but you should do a little cleanup - in your dev environment, change this yourself and commit it to your relevant branch so it doesn't show up as cruft here.